Content/Buffs/BloodclotDebuff.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace PathOfTerraria.Content.Buffs;
 
-public class BloodclotDebuff : ModBuff
+public sealed class BloodclotDebuff : ModBuff
 {
 	public override void SetStaticDefaults()
```

Comment:
Is there a strong reason for this and MoltenShieldBuff being sealed? It doesn't matter much, but the only thing this'd change is stopping others from inheriting for some reason; I don't see a benefit to that (and it's extremely unlikely there's a downside either, realistically, but still.)

Reply:
I think I just wanted to follow up on what the other buffs did. For some reason, some other content pieces are also sealed. I think it's rather unlikely we end up doing abstraction between buffs and such.

Reply:
Huh, okay then. If somehow some cross-mod wants to inherit we could always just undo it, easy change.